openshift-helm-charts/charts
# Add support for Red Hat postgresql imagestreams

This Helm chart contains all Red Hat PostgreSQL imagestreams

Please review it and verify it before merging.

The output from verifier is:

```bash
results:
    - check: v1.0/required-annotations-present
      type: Mandatory
      outcome: PASS
      reason: All required annotations present
    - check: v1.0/chart-testing
      type: Mandatory
      outcome: FAIL
      reason: 'error loading config file "/.kube/config": open /.kube/config: permission denied'
    - check: v1.0/contains-test
      type: Mandatory
      outcome: FAIL
      reason: Chart test files do not exist
    - check: v1.0/contains-values-schema
      type: Mandatory
      outcome: FAIL
      reason: Values schema file does not exist
    - check: v1.1/has-kubeversion
      type: Mandatory
      outcome: PASS
      reason: Kubernetes version specified
    - check: v1.0/has-readme
      type: Mandatory
      outcome: PASS
      reason: Chart has a README
    - check: v1.0/not-contain-csi-objects
      type: Mandatory
      outcome: PASS
      reason: CSI objects do not exist
    - check: v1.0/not-contains-crds
      type: Mandatory
      outcome: PASS
      reason: Chart does not contain CRDs
    - check: v1.0/contains-values
      type: Mandatory
      outcome: FAIL
      reason: Values file does not exist
    - check: v1.0/helm-lint
      type: Mandatory
      outcome: PASS
      reason: Helm lint successful
    - check: v1.1/images-are-certified
      type: Mandatory
      outcome: PASS
      reason: No images to certify
    - check: v1.0/is-helm-v3
      type: Mandatory
      outcome: PASS
      reason: API version is V2, used in Helm 3
    - check: v1.0/signature-is-valid
      type: Mandatory
      outcome: SKIPPED
      reason: 'Chart is not signed : Signature verification not required'

```
The Chart itself does not have any test suite. values.yaml file is not needed at all.
